_data/navbars/maintenance.yml

```diff
@@ -11,6 +11,11 @@ section:
     path: /maintenance/openshift-upgrade
   - title: OpenStack
     path: /maintenance/openstack-upgrade
+- title: Install image options
```

Comment:
Let's make verb the same to match tasks underneath parent...my fault for not seeing this in first pass. WDYT?

Deploy image options